app/components/primer/avatar_stack_component.rb

```diff
@@ -40,14 +43,17 @@ class AvatarStackComponent < Primer::Component
     # @param align [Symbol] <%= one_of(Primer::AvatarStackComponent::ALIGN_OPTIONS) %>
     # @param tooltipped [Boolean] Whether to add a tooltip to the stack or not.
     # @param body_arguments [Hash] Parameters to add to the Body. If `tooltipped` is set, has the same arguments as <%= link_to_component(Primer::Tooltip) %>.
+    #   The default tag is <%= pretty_value(Primer::AvatarStackComponent::DEFAULT_BODY_TAG) %> but can be changed using `tag:`
+    #   to <%= one_of(Primer::AvatarStackComponent::BODY_TAG_OPTIONS).downcase %>
```

Comment:
What is the `downcase` for here?

Reply:
`one_of` renders `One of _____` because we most often use use it on it's own.
In this case, we're combining it with another sentence so we'd want to downcase.

Reply:
Oh! actually I just noticed there's a `lower` argument on the `one_of` method 🙏. I'll use that :)